main.go

```diff
@@ -146,40 +145,41 @@ func handle(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 			return
 		} else if errors.Is(err, fs.ErrNotExist) {
-			// go build
-			args := []string{"build", "-o", filepath.Join(output, "main.wasm")}
+			// go run
+			args := []string{"run", "-exec", "cp"}
```

Comment:
Let's leave a comment

* What is `-exec` (just saying `go help run` would be fine)
* Why we cannot use `GOBIN` + `go install`